backend/api.js

```diff
@@ -0,0 +1,135 @@
+const express = require('express')
+const app = express()
+const port = 3001;
+const cors = require('cors')
+const parser = require('body-parser')
+const knex = require('knex')(require('./knexfile.js')['development']);
+//cors policy update
+app.use(cors())
+app.use(parser.json())
+
+//Root for api server connection testing
+app.get('/', (request, response) => {
+  response.status(200).send('Successful')
+})
```

Comment:
This is an interesting idea.  We may not 100% NEED to have a route specifically to test the server's connection, since a failure to render a response from any route would indicate a connection problem (or a missing response.send() or response.sendStatus() on the route in question - although if THIS is the eventuality that this test route provides for, it is unlikely to be useful past the point of finishing the server's routes' code).